core/mm/core_mmu.c

```diff
@@ -286,8 +286,13 @@ static struct tee_mmap_region *find_map_by_pa(unsigned long pa)
 	struct tee_mmap_region *map = get_memory_map();
 
 	while (!core_mmap_is_end_of_table(map)) {
+		/* Skip unmapped regions */
+		if (!(map->attr & TEE_MATTR_VALID_BLOCK))
+			goto next_region;
+
 		if (pa >= map->pa && pa <= (map->pa + map->size - 1))
```

Comment:
I would prefer adding `(map->attr & TEE_MATTR_VALID_BLOCK) &&` to this expression or possibly turning the loop into a for loop so `continue` would do the right thing.

Reply:
Hi @jenswi-linaro 
Patch updated, thanks for the review.